workflows/convert_input_format.py

```diff
@@ -0,0 +1,147 @@
+# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+
+import os
+import platform
+import shutil
+import subprocess
+
+from eplaunch.workflows.base import BaseEPLaunchWorkflow1, EPLaunchWorkflowResponse1
+
+
+class ConvertInputFormatWorkflow(BaseEPLaunchWorkflow1):
+
+    def name(self):
+        return "ConvertInputFormat-${CMAKE_VERSION_MAJOR}.${CMAKE_VERSION_MINOR}.${CMAKE_VERSION_PATCH}"
+
+    def context(self):
+        return "EnergyPlus-${CMAKE_VERSION_MAJOR}.${CMAKE_VERSION_MINOR}.${CMAKE_VERSION_PATCH}-${CMAKE_VERSION_BUILD}"
+
+    def description(self):
+        return "Run ConvertInputFormat"
+
+    def get_file_types(self):
+        return ["*.idf", "*.epJSON"]
+
+    def get_output_suffixes(self):
+        return [".idf", ".epJSON"]
+
+    def get_extra_data(self):
+        return {"Hey, it's extra": "data"}
+
+    def get_interface_columns(self):
+        return []
+
+    def main(self, run_directory, file_name, args):
+        if 'workflow location' in args:
+            energyplus_root_folder, _ = os.path.split(args['workflow location'])
+            if platform.system() == 'Windows':
+                convertinputformat_binary = os.path.join(energyplus_root_folder, 'ConvertInputFormat.exe')
+            else:
+                convertinputformat_binary = os.path.join(energyplus_root_folder, 'ConvertInputFormat')
+            if not os.path.exists(convertinputformat_binary):
+                return EPLaunchWorkflowResponse1(
+                    success=False,
+                    message="ConvertInputFormat binary not found: {}!".format(coeffconv_binary),
+                    column_data=[]
+                )
+        else:
+            return EPLaunchWorkflowResponse1(
+                success=False,
+                message="Workflow location missing: {}!".format(args['worflow location']),
+                column_data=[]
+            )
+
+        original_with_path = os.path.join(run_directory, file_name)
+        converted_file_no_ext, original_ext = os.path.splitext(original_with_path)
+        if original_ext.lower() == '.idf':
+            converted_file_with_path = converted_file_no_ext + '.epJSON'
+        elif original_ext.lower() == '.epjson':
+            converted_file_with_path = converted_file_no_ext + '.idf'
+        else:
+            return EPLaunchWorkflowResponse1(
+                success=False,
+                message="Invalid extension {} on file: {}!".format(original_ext, original_with_path),
+                column_data=[]
+            )
+        	
+
+        if os.path.exists(original_with_path) and os.path.exists(convertinputformat_binary) and os.path.exists(run_directory):
+
+            # execute utility
+            command_line_args = [convertinputformat_binary, original_with_path]
+            process = subprocess.run(
```

Comment:
I _think_ for consistency, I would like this changed to the workflow helper function for executing subprocesses.  I realize we don't get much output from this tool, but in the future maybe we would get a little more?